### DISCUSSION TITLE
Joomla 4 RC 1

### DISCUSSION THREAD

**Discussion by 4digitalworld** [joomla/joomla-cms#32835] [General]
how far is now from RC1? 

a stable version of Bootstrap 5 will be released in a month.

**Comment by laurentgarin** [comment]
Yes it's true that it would be nice to have just an idea of when the RC1 will be released?

Thanks.

**Comment by alikon** [comment]
As soon as we can solve all the Release Blockers....

**Reply by laurentgarin** [reply]
OK thanks, I have only modest knowledge in development otherwise I will have helped for a long time ...

**Comment by dgrammatiko** [comment]
How about testing the PR for the latest Bootstrap Beta https://github.com/joomla/joomla-cms/pull/32827 ?

You know if PRs are not tested, they don't get merged and there is no progress...

**Reply by laurentgarin** [reply]
yes i understand but frankly, it would be really with pleasure if I knew how to do, I would have to try again ...
The github development environment is not very simple when you don't know much

**Reply by HLeithner** [reply]
We create ready to install/upgrade packages for each PR you can find it on the github pr site under download. For the mentioned or it's  
https://ci.joomla.org/artifacts/joomla/joomla-cms/4.0-dev/32827/downloads/41153/

**Reply by laurentgarin** [reply]
OK thanks, i am going to see that :-)

**Reply by laurentgarin** [reply]
OK i have test this and it is OK, what i need to do now ? Thanks.

**Reply by wojsmol** [reply]
@WebAtelier47 Go to https://issues.joomla.org/tracker/joomla-cms/32827 login with your github account and mark tests results.

**Reply by dgrammatiko** [reply]
@WebAtelier47 thanks for the test

**Reply by laurentgarin** [reply]
@dgrammatiko It's a pleasure to be able to help at my modest level :-)

**Comment by 4digitalworld** [comment]
A stable version of Bootstrap 5 has been released [https://github.com/twbs/bootstrap/releases/tag/v5.0.0](https://github.com/twbs/bootstrap/releases/tag/v5.0.0)

I think it was the right decision when there was a transition from B4 to B5, because Joomla is still not even a RC
